### PR TITLE
Base image update

### PR DESCRIPTION
**Pull request by atselousov**
What has changed:
- base image. So, python has the version 3.7 now;
- added some utils which members of the research team always add by hands;
- deleted the installation of some packages.

I've decided to use a different base image because of this issue:  
https://neuromation.slack.com/archives/CASUYPPK6/p1589448924103700

With the official `pytorch` image there is no error occurred during the following simple test:
Screenshot from 2020-07-16 12-32-23

What do you folks think?

### CODE REVIEW

**Comment by yartem** [comment]
Note: I fixed linter for tests and pushed to your branch

**Comment by atselousov** [comment]
I also tried to add `nvtop` to the base image, but I decided not to do it because of this [issue](https://github.com/Syllo/nvtop/issues/45).

**Comment by yartem** [comment]
Thank you very much for this PR! :tada:
- LGTM as it's better to have later versions of python and torch.
- I'm afraid many tests on pip deps will fail, will try to fix them.
- Is `assert cuda.is_available()` a guarantee that torch can see GPU?
- Also, it's better to test that tensorflow works on GPU

**Comment by atselousov** [comment]
`assert cuda.is_available()`

I also tried to run an inference with a small model on GPU. It worked and there was no error about drivers version.

But of course, I'll test the image in the proper way a little bit later.

**Comment by atselousov** [comment]
Tests failed on `test_wandb` because of a incorrect token (

File: targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# python        3.7       (apt)
 # jupyter       latest    (pip)
 # jupyterlab    latest    (pip)
-# pytorch       1.2.0     (pip)
+# pytorch       1.5.1     (pip)
 # tensorflow    2.0.0-rc1 (pip)
 # ==================================================================
 
-FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+FROM pytorch/pytorch:1.5.1-cuda10.1-cudnn7-devel
```
Comment (yartem) [inline targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:11]: Big change :wink:

Comment (mariyadavydova) [reply targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:11]: Wait, I overlooked this. What about Tensorflow? Will it work on this image as it was?

Comment (atselousov) [reply targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:11]: I don't see any reason why TF would not work. Drivers the same, just version of Cuda toolkit has been changed.

**Comment by atselousov** [comment]
I have tested this image with my current pose estimation project. Everything worked fine, `torch==1.5.1` works.

But I still have problems with the time of computations with half-precision.

**Comment by atselousov** [comment]
I also want to mention that `WORKSPASE` has also been changed. Now `WORKSPASE=/workspase`. If there could be any problems because of it, just change it or ask me to do it.

**Comment by atselousov** [comment]
I've updated TF version. `tensorflow==2.2.0` works file with 10.1 cudatoolkit.

File: targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile (hunk 1 above)
Comment (yartem) [reply targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:11]: Indeed, itself it is build from [`nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04`](https://github.com/pytorch/pytorch/blob/fe415589a964697a78186189bd4815541f853b47/docker/pytorch/Dockerfile#L1)

**Comment by yartem** [comment]
> Now WORKSPASE=/workspase

For now, I changed it to `/` as it is in the older image. Later, we'll change it to `WORKDIR /project` that's being used by our Template.

**Pull request merged**